### DISCUSSION TITLE
RevealJS gradient background

### DISCUSSION THREAD

**Discussion by sagikazarmark** [quarto-dev/quarto-cli#6873] [Feature Requests]
### Description

Add support for RevealJS gradient backgrounds: https://revealjs.com/backgrounds/#gradient-backgrounds.

I tried the following, but it doesn't seem to work:

```markdown
# Demo {data-background-gradient="linear-gradient(45deg, blue, red)"}
```

(Tried `background-gradient` as well)

**Comment by mcanouil** [comment]
FYI: This is from Reveal.js [4.4.0](https://github.com/hakimel/reveal.js/releases/tag/4.4.0)

Current pre-release version embedded in Quarto is 4.3.1.
Dependencies are updated once per version/release, so support for that will likely arrive in Quarto 1.5.

(cc @cderv)

**Reply by sagikazarmark** [reply]
Thanks @mcanouil!

Any chance it would get upgraded to the latest version?

**Reply by cscheid** [reply]
As @mcanouil answered,

> support for that will likely arrive in Quarto 1.5.

**Reply by mcanouil** [reply]
As I said, dependencies are updated once per Quarto release.
There was already an update.

A dependency update can break features, thus there is work to be done for each update and currently there are more important work ongoing to take the risk of breaking things.

**Reply by sagikazarmark** [reply]
The question rather referred to the version you upgrade to, not when it happens. As of today the latest version is 4.6.0, 4.4.0 is almost a year old now.

**Reply by mcanouil** [reply]
It will likely be upgraded to the latest stable version available at the time it will happen.

**Comment by cderv** [comment]
Quarto 1.6 pre-release has an update Revealjs, and so background gradient is available
- https://prerelease.quarto.org/docs/presentations/revealjs/#gradient-background
